### ISSUE TITLE
In-Game: static_get() silently crashes the runner when "undefined" is the expected result for that struct

### ISSUE THREAD

**Issue by tinkerer-red** [YoYoGames/GameMaker-Bugs#8048]
### Description

Some structs will crash with out error when a `static_get` is used on them. I have found two examples currently.

### Expected Change

`static_get()` should return undefined as it does with the default struct

### Steps To Reproduce

```gml
static_get(global) // instanceof returns string "Object" on this struct
// Additionally this also fails
var effect = fx_create("_filter_colourise");
var _test = static_get(effect) // instanceof returns null value `undefined` on this struct
```

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.8.1 (Monthly), 2024.1100 (Betas)

### Which platform(s) are you seeing the problem on?

Windows

### Sample Package Attached?

- [X] I have attached my bug package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by rwkay** [comment]
Fixed in 2024.11

**Comment by sihammill** [comment]
Fix verified in IDE v2024.1100.0.653  Runtime v2024.1100.0.679
